NEWS.md

```diff
@@ -15,7 +34,7 @@ Fixes:
  * [OKAPI-1031](https://issues.folio.org/browse/OKAPI-1031) "apt-get install okapi" should recreate `/var/lib/okapi`
 
 Other:
- * [OKAPI-1035](https://issues.folio.org/browse/OKAPI-1035) Upgrade to Vert.x 4.1.4 OKAPI-1035
+ * [OKAPI-1035](https://issues.folio.org/browse/OKAPI-1035) Upgrade to Vert.x 4.1.4 [OKAPI-1035](https://issues.folio.org/browse/OKAPI-1035)
```

Comment:
`OKAPI-1035` is already at the beginning of the line, can be removed from the end of line